apps/camera/js/controllers/preview-gallery.js

```diff
@@ -399,6 +399,7 @@ PreviewGalleryController.prototype.updateThumbnail = function() {
 
     // If it is a video we can create a thumbnail from the poster image
     blob = media.poster.blob;
+    media = media.poster;
```

Comment:
This might be cleaner if you used a different variable name.
Since media refers to the same object as this.thumbnailItem, I wonder if you need to update this.thumbnailItem in this case, too.